src/screens/main/ai/bookRecommendation/BookRecommendationScreen.tsx

```
@@ -0,0 +1,25 @@
+
+  const MOCK_BOOKS = [
+    { id: 1, image: require("@/assets/img_test-book_default.png") },
+    { id: 2, image: require("@/assets/img_test-book_default.png") },
+    { id: 3, image: require("@/assets/img_test-book_default.png") },
+    { id: 4, image: require("@/assets/img_test-book_default.png") },
+    { id: 5, image: require("@/assets/img_test-book_default.png") },
+  ];
+  return (
+    <S.Container>
+      <S.TitleBox>
+        <Typography font="medium28" color="defaultBlack" children={`${userName}님을 위한 추천 책`} />
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)